build_tools/cmake/build_all.sh

```diff
@@ -48,6 +48,10 @@ declare -a CMAKE_ARGS=(
   "-DPYTHON_EXECUTABLE=${IREE_PYTHON3_EXECUTABLE}"
 
   "-DIREE_TARGET_BACKEND_WEBGPU=${IREE_TARGET_BACKEND_WEBGPU}"
+
+  # The libzstd on Ubuntu 18.04 doesn't work with the current LLVM.
```

Comment:
Wait why is this only just now coming up? The version of llvm on the 18.04 images shouldn't have changed

Reply:
It's the integrated LLVM (`third_party/llvm`) can't built with libztd-dev on Ubuntu 18.04. Updated the comments.